spectral_cube/dask_spectral_cube.py

```diff
@@ -58,6 +68,26 @@ def wrapper(self, *args, **kwargs):
     return wrapper
 
 
+def add_save_to_tmp_dir_option(function):
```

Comment:
should this also append the docstring, instead fo c&p'ing it below?

Reply:
Perhaps, but making it do it properly and insert it into the right section is a bit of a pain. It wasn't that hard to copy/paste ;)

Reply:
fair enough - I like the auto-add-docstring approach because it means there's a single point for maintenance, but it does make it harder to find the docs at not-runtime.